roles/fact_regular_users/tasks/main.yml

```diff
@@ -30,6 +30,6 @@
 - name: set fact regular_users
   set_fact:
     fact_regular_users: "{{ getent_passwd|dict2items|to_json|from_json|
-    json_query('[?to_number(value[1]) >`999` && starts_with(value[4], `/home/`) 
+    json_query('[?to_number(value[1]) >`999` && starts_with(value[4], `/home/`) && key!=`ubuntu`
```

Comment:
@tsmeele In stead of always removing user 1000, there is now an explicit check to exclude the user named 'ubuntu'.

Reply:
Additional info: apparently there is a `/home/ubuntu` directory present on the workspaces. We cannot however remove that check, because it helps filtering out the `nobody` user, and I think in general it is save to say that we do not wish to include users without a home directory.